zend_abstract_interface/zai_string/string.h

```diff
@@ -1,35 +1,53 @@
 #ifndef ZAI_STRING_H
 #define ZAI_STRING_H
 
+// Include this before standard headers, or you may get strange errors like:
+// error: unknown type name 'siginfo_t'
+#include <Zend/zend.h>
+
+#include <zai_assert/zai_assert.h>
+
 #include <stdbool.h>
 #include <stddef.h>
 #include <string.h>
 
-#include <Zend/zend.h>
+// never write, only read
+extern char *ZAI_STRING_EMPTY_PTR;
```

Comment:
What's this? It seems unused?

Reply:
Ah, probably an accidental merge when I picked a fix from zai-string, I'll remove it.